scripts/main/contextMenu.js

```diff
@@ -416,6 +416,7 @@ contextMenu.photoMore = function (photoID, e) {
 	let items = [
 		{ title: build.iconic("fullscreen-enter") + lychee.locale["FULL_PHOTO"], visible: !!showFull, fn: () => window.open(photo.getDirectLink()) },
 		{ title: build.iconic("cloud-download") + lychee.locale["DOWNLOAD"], visible: !!showDownload, fn: () => photo.getArchive([photoID]) },
+		{ title: build.iconic("cloud-download") + lychee.locale["QR_CODE"], fn: () => photo.qrCode([photoID]) },
```

Comment:
"cloud-download" => you may want to use another icon name.
This should help you: view-source:https://raw.githubusercontent.com/LycheeOrg/Lychee-front/master/images/iconic.svg

Reply:
I searched through them, and there are no QR code icons.

Reply:
maybe `grid-two-up` ?

Reply:
https://github.com/LycheeOrg/Lychee-front/pull/287/commits/05868611062961cb72e9a0e108e8d784f47b1b24